src/flake8_comprehensions.py

```diff
@@ -176,6 +180,94 @@ def run(self):
                         self.messages["C408"].format(type=node.func.id),
                         type(self),
                     )
+
+                elif (
+                    node.func.id in {"list", "reversed"}
+                    and num_positional_args > 0
+                    and isinstance(node.args[0], ast.Call)
+                    and isinstance(node.args[0].func, ast.Name)
+                    and node.args[0].func.id == "sorted"
+                ):
+                    msg = self.messages["C413"]
+                    if node.func.id == "reversed":
+                        reverse_value = (
+                            [False]
+                            if not node.args[0].keywords
+                            else [
+                                kw.arg == "reverse"
+                                and (
+                                    isinstance(kw.value, ast.NameConstant)
+                                    and kw.value.value
+                                    or isinstance(kw.value, ast.Num)
+                                    and kw.value.n != 0
+                                )
+                                for kw in node.args[0].keywords
+                                if isinstance(kw.value, (ast.NameConstant, ast.Num))
+                            ]
+                        )
+                        if reverse_value:
+                            msg = msg.rstrip(
+                                "."
+                            ) + " - use sorted(..., reverse={!r}).".format(
+                                not reverse_value[0]
+                            )
+                        else:
+                            msg = (
+                                msg.rstrip(".")
+                                + " - toggle reverse argument to sorted()."
+                            )
```

Comment:
I couldn't follow this so I've rewritten it with a `for` loop and an extra message piece.